src/Model/GServer.php

```diff
@@ -1623,6 +1627,80 @@ private static function getNomadVersion(string $url): string
 		return $data ?? '';
 	}
 
+	private static function getZotData(string $url, array $serverdata): array
+	{
+		$curlResult = DI::httpClient()->get($url, 'application/x-zot+json');
+		if (!$curlResult->isSuccess()) {
+			return $serverdata;
+		}
+		$json = $curlResult->getBodyString();
+		$data = json_decode($json, true);
+		if (empty($data)) {
+			return $serverdata;
+		}
+
+		if (!empty($data['site'])) {
+			$serverdata = self::getFromZotData($data['site'], $serverdata);
+		} else {
+			$serverdata = self::getFromZotData($data, $serverdata);
+		}
+		return $serverdata;
+	}
+
+	private static function getFromZotData(array $data, array $serverdata): array
+	{
+		if (!empty($data['version'])) {
+			$serverdata['version'] = $data['version'];
+		}
+
+		if (!empty($data['openWebAuth'])) {
+			$serverdata['openwebauth'] = $data['openWebAuth'];
+		}
+
+		if (!empty($data['authRedirect'])) {
+			$serverdata['authredirect'] = $data['authRedirect'];
+		}
+
+		if (!empty($data['sitename'])) {
+			$serverdata['site_name'] = $data['sitename'];
+		}
+
+		if (!empty($data['about'])) {
+			$serverdata['info'] = $data['about'];
+		}
+
+		if (empty($serverdata['info']) && !empty($data['location'])) {
+			$serverdata['info'] = $data['location'];
+		}
+
+		if (!empty($data['project']) && in_array($data['project'], ['hubzilla', 'streams', 'osada', 'mistpark', 'roadhouse', 'zap'])) {
+			$serverdata['platform'] = $data['project'];
+		}
+
+		if (!empty($data['accounts'])) {
+			$serverdata['registered-users'] = $data['accounts'];
+		}
+
+		if (!empty($data['register_policy'])) {
+			switch ($data['register_policy']) {
+				case 'open':
+					$serverdata['register_policy'] = Register::OPEN;
+					break;
+				case 'closed':
+					$serverdata['register_policy'] = Register::CLOSED;
+					break;
+				case 'approve':
+					$serverdata['register_policy'] = Register::APPROVE;
+					break;
+				default:
+					echo $data['register_policy'] . "\n";
```

Comment:
`echo`?

Reply:
Is removed in https://github.com/friendica/friendica/pull/14182

Reply:
I saw this later, thanks!